src/components/pages/__tests__/InntektStep.test.tsx

```diff
@@ -0,0 +1,734 @@
+import { render, screen, fireEvent } from '@testing-library/react'
```

Comment:
inntektStep linje 116-120 og 139-143 (+ 4 andre steder):
Array.from bør helst ikke være direkte i render metoden. Det er en ganske tung metode som vil kjøre for hver re-render og påvirker ytelse negativt.
Den kan f.eks heller generer din array én gang å ha den lagret i en memo, så kan du bruke .map i render.